v2/cn/getting-started.md

```diff
@@ -55,46 +71,62 @@ If you're familiar with [Nginx](nginx.html) configuration, it should look very
 familiar to you. [OpenResty](openresty.html) is just an enhanced version of
 [Nginx](nginx.html) by means of addon modules anyway. You can take advantage
 of all the exisitng goodies in the [Nginx](nginx.html) world.
+如果你很了解 Nginx 的布局，上面的内容你会觉得很熟悉。OpenResty 只是一个借助于插件模块来实现的 Nginx 加强版。你可以利用 Nginx 世界里所有的好东西。
 
 ## Start the [Nginx](nginx.html) server
+## 开始使用 Nginx 服务器
+
 Assuming you have installed [OpenResty](openresty.html) into `/usr/local/openresty` (this
 is the default), we make our `nginx` executable of our [OpenResty](openresty.html) installation
 available in our `PATH` environment:
+假如你已经将 OpenResty 安装到 /usr/local/openresty （这是默认值），那么 Nginx 就可以在 PATH 环境下的 OpenResty 配置里运行了：
```

Comment:
Please do not remove backticks and do not change the letter case of `nginx`, for example.